splice/ingest.py

```diff
@@ -205,6 +129,8 @@ def remove_unserializable_data(tile):
                     if not env.is_test:
                         conn.execute("LOCK TABLE tiles; LOCK TABLE adgroups; LOCK TABLE adgroup_sites;")
 
+                    if is_compact and not populate_image_uris(t, assets):
+                        continue
```

Comment:
Can you please add a TODO comment here and reference the bug below?

This is debt we'll need to tackle. I opened a bug here: [Bug 1169302 - Collect and return Tile ingestion errors](https://bugzilla.mozilla.org/show_bug.cgi?id=1169302)